### ISSUE TITLE
One Click Installer

### ISSUE THREAD

**Issue by Tobe2d** [Sanster/IOPaint#441]
Any news on when we can update the one click installer?

**Comment by Sanster** [comment]
hi, I have already uploaded the installer for IOPaint. You can download it from the original order page through the email from lemon squeeze. If you can't find the original email, you can log in to [lemon squeeze](https://www.lemonsqueezy.com/) using your mailbox to find the order.

<img width="490" alt="image" src="https://github.com/Sanster/IOPaint/assets/3998421/6054ddbe-f8f3-40e3-a51f-ab959693f391">

**Comment by Tobe2d** [comment]
Thanks for the update @Sanster 

Could you please guide me on 2 things:

1 - how can I download the new supported models? as in the olde version we select it and it auto start downloading but now I cant find SDXL or anytext models and dont know what to download exactly.

2 - how can I make win_start.bat autostart the GUI in web browser? as of now win_config.bat do open the web browser automaticly but win_start.bat does not auto open.


Thanks in advanced

**Comment by Sanster** [comment]
> 1 - how can I download the new supported models? as in the olde version we select it and it auto start downloading but now I cant find SDXL or anytext models and dont know what to download exactly.

Because IOPaint can support users to use any SD/SDXL model, I have modified the method of model selection. I have added some documentation [here](https://www.iopaint.com/install/windows_1click_installer#model-select).

> 2 - how can I make win_start.bat autostart the GUI in web browser? as of now win_config.bat do open the web browser automaticly but win_start.bat does not auto open.

This feature is currently not supported. It's a good suggestion, I will add this feature in the later version.

**Comment by sebaxakerhtc** [comment]
@Sanster 
Hi. I have paid for lama cleaner a year ago.
There's no updates, so my question is "Should I pay again or there's an option to download latest version?"

**Comment by Sanster** [comment]
> @Sanster Hi. I have paid for lama cleaner a year ago. There's no updates, so my question is "Should I pay again or there's an option to download latest version?"

No, you can find the new installation package on the original download page.

<img width="495" alt="image" src="https://github.com/Sanster/IOPaint/assets/3998421/892511b2-1895-418e-af57-9ecdd2aa407f">

**Comment by Saaalih2g** [comment]
ماذا يعني هذا البريد

في أربعاء، 12 يونيو، 2024 في 6:16 ص، كتب Qing ***@***.***>:
